frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -790,8 +790,9 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 			print_settings: print_settings,
 			landscape: landscape,
 			filters: this.get_filter_values(),
-			data: custom_format ? this.data : this.get_data_for_print(),
```

Comment:
Why pass get_data_for_print to custom formats?

Reply:
Because the issue was that we don't have the filtered/ordered data in the custom formats. The "default" data should be the filtered/ordered and the original data should be accessible but not default.

Reply:
@netchampfaris